playbooks/roles/hammer_devel/tasks/hammer_install.yml

```diff
@@ -7,6 +7,12 @@
     remote: upstream
   with_items: "{{ hammer_devel_repositories }}"
 
+- name: 'Add fork remotes to cloned repositories'
```

Comment:
@ekohl Hmm... it doesn't actually appear to be that simple. I'm not sure you can use `when` with a command directly. I've been trying to search through the ansible docs but haven't come across anything.